### PR TITLE
Save/load tracing.

### PR DESCRIPTION
**Pull request by polina-c**

### CODE REVIEW

File: packages/devtools_app/lib/src/screens/memory/panes/tracing/tracing_data.dart [hunk 1]
```diff
@@ -3,18 +3,28 @@
...
+import '../../../../shared/primitives/encoding.dart';
+import '../../../../shared/primitives/simple_items.dart';
 import '../../../../shared/primitives/utils.dart';
 import '../../../../shared/table/table_data.dart';
 import '../../../profiler/cpu_profile_model.dart';
 import '../../../profiler/cpu_profile_transformer.dart';
 
+@visibleForTesting
+enum TracedClassJson {
+  cls,
```
Comment (kenzieschmoll) [inline packages/devtools_app/lib/src/screens/memory/panes/tracing/tracing_data.dart:21]: nit: how about clazz instead so we can avoid abbreviated names

File: packages/devtools_app/test/test_infra/scenes/memory/default.dart [hunk 2]
```diff
@@ -182,5 +182,10 @@ class MemoryDefaultScene extends Scene {
     await tester.pumpAndSettle();
   }
 
+  Future<void> goToTraceTab(WidgetTester tester) async {
```
Comment (kenzieschmoll) [inline packages/devtools_app/test/test_infra/scenes/memory/default.dart:185]: where is this used in this PR?

**Review by kenzieschmoll — APPROVED** [review]
just a couple comments

File: packages/devtools_app/test/test_infra/scenes/memory/default.dart (hunk 2 above)
Comment (polina-c) [reply packages/devtools_app/test/test_infra/scenes/memory/default.dart:185]: It is not used. Will use it in next PR for testing.

**Pull request merged**